### ISSUE TITLE
[Q] Is it possible to use this framework to author standalone web apps rather than addons

### ISSUE THREAD

**Issue by mjbedford** [enuchi/React-Google-Apps-Script#175]
Just found this repo, an amazing set of work thank you. I came across it as I am trying to develop a Google Apps Script based web app that uses Google Sheets and Google Drive in the backend and wanted to know if it was possible to use this framework to develop this type of solution.

**Comment by enuchi** [comment]
Yes it is possible. Search the other issues in this repo, there are some discussions on how to achieve this.

Note there are some limitations to webapps. You won't be able to use react-router or anything like that to build multiple pages because webapps don't support the standard browser history API. But you can build standalone pages using this framework.

**Comment by mjbedford** [comment]
Thank you, I'll take a look through the issues, shame about react-router, that is my goto strategy for dev, I'll have to think a bit laterally.
